pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -892,7 +897,38 @@ void run() throws PulsarAdminException {
                 throw new ParameterException("--bundle and --bundle-type are mutually exclusive");
             }
             bundle = bundleType != null ? bundleType.toString() : bundle;
-            getAdmin().namespaces().splitNamespaceBundle(namespace, bundle, unload, splitAlgorithmName);
+            getAdmin().namespaces().splitNamespaceBundle(
+                    namespace, bundle, unload, splitAlgorithmName, splitBoundaries);
+        }
+    }
+
+    @Parameters(commandDescription = "Get the positions for one or more topic(s) in a namespace bundle")
+    private class GetTopicHashPositions extends CliCommand {
+        @Parameter(description = "tenant/namespace", required = true)
+        private java.util.List<String> params;
+
+        @Parameter(
+                names = { "--bundle", "-b" },
+                description = "{start-boundary}_{end-boundary} format namespace bundle",
+                required = false)
+        private String bundle;
+
+        @Parameter(
+                names = { "--topic-list",  "-tl" },
+                description = "The list of topics to get posisions in this bunel",
```

Comment:
```suggestion
                description = "The list of topics to get position in this bundle",
```

Reply:
Can these topic be partitioned topics?

Reply:
Yes, both non-partitioned topic and partitioned-topics